instagram/spec/helpers/posting.rb

```diff
@@ -0,0 +1,17 @@
+def sign_in
+  User.create(user_name: "tester", email: "test@live.com", password: "password", password_confirmation: "password")
```

Comment:
Metrics/LineLength: Line is too long. [115/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)